sitemap.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+   <url>
+      <loc>http://matplotlib.org/stable/</loc>
```

Comment:
Note these should really have a `<last-modified>` tag, but I think they will work without.